src/metadata_client.ts

```diff
@@ -216,8 +216,24 @@ export class MetadataClient {
       return this.controller.$q.when(operators);
     }
     let tagsCopy = [... this.queryModel.target.tags];
-    if (segment.type === "value") {
-      tagsCopy = tagsCopy.splice(0, tagsCopy.length - 1);
+    if (segment.type === "value" || (segment.type === "key")) {
```

Comment:
this fixes an issue where tag value lookup does not work for tag/values that are in the middle of a query